keyboards/okuura/goaisatsu/readme.md

```diff
@@ -0,0 +1,17 @@
+# goaisatsu
+
+![goaisatsu](https://i.imgur.com/rd0MFFlh.jpg)
+
+*Simple JIS Keyboard.*
+
+* Keyboard Maintainer: [okuura](https://github.com/okuura)
+* Hardware Supported: goaisatsuPCB, ProMicro
+* Hardware Availability: [Booth](https://okuura.booth.pm/items/4980888)
+
+Make example for this keyboard (after setting up your build environment):
+
+    make goaisatsu:default
+
+Flashing example for this keyboard:
+
+    make goaisatsu:default:flash
```

Comment:
Could you add instructions on how to enter the bootloader, to flash new firmware?  This is needed, as a new user may not be aware of how to enter to bootloader to flash new firmware.

Eg: 
```markup
## Bootloader

Enter the bootloader in 3 ways:

* **Bootmagic reset**: Hold down the key at (0,0) in the matrix (usually the top left key or Escape) and plug in the keyboard
* **Physical reset button**: Briefly press the button on the back of the PCB - some may have pads you must short instead
* **Keycode in layout**: Press the key mapped to `QK_BOOT` if it is available
```

Reply:
@drashna Thanks for your review. I added a description of the bootloader.